src/ragger/backend/speculos.py

```diff
@@ -139,34 +149,30 @@ def _screenshot_equal_with_crop(self, path: Path, bytes: BytesIO, crop: Crop = N
         else:
             return screenshot_equal(f"{path}", bytes)
 
-    def _compare_screenshot_with_timeout(self,
-                                         path: Path,
-                                         timeout_s: float = 5.0,
-                                         crop: Crop = None):
+    def _compare_snap_with_timeout(self, path: Path, timeout_s: float = 5.0, crop: Crop = None):
         start = time()
         now = start
         while not (now - start > timeout_s):
             screenshot = self._client.get_screenshot()
-            if self._screenshot_equal_with_crop(path, BytesIO(screenshot), crop):
+            if self._snap_equal_with_crop(path, BytesIO(screenshot), crop):
                 return True
             now = time()
         return False
 
     def _compare_snaps(self, path: Path, test_case_name: Path, last_img_idx: int = 0) -> bool:
 
-        snapshot_golden_path = path / "snapshots" / self._firmware.device / test_case_name
```

Comment:
Why the hassle of renaming these variables just to make them a bit shorter ?

Reply:
the idea is that we have `screenshot`, `snap` and `snapshot` naming, I think it would be better to harmonize on one name?